src/Microsoft.HttpRepl/UserProfile/UserProfileDirectoryProvider.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.HttpRepl.UserProfile
+{
+    public class UserProfileDirectoryProvider : IUserProfileDirectoryProvider
+    {
+        public string GetUserProfileDirectory()
+        {
+            bool isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
Do we need to handle OSPlatform.OSX and OSPlatform.Linux here?

Reply:
They both act the same way (using the HOME environment variable). Windows is the only weirdo of the supported platforms at the moment, as far as I can tell.